### PR TITLE
duplicate: allow for smaller width now that panel are resizable.

### PR DESCRIPTION
**Pull request by TurboGit**

### CODE REVIEW

**Comment by TurboGit** [comment]
@Nilvus : you may want to review this. Thanks.

File: data/themes/darktable.css [hunk 1]
```diff
@@ -876,8 +876,8 @@ entry
 #left #lib-plugin-ui scrolledwindow entry /* This will avoid having too much height on entry items in clone manager without touching other parts of the UI */
 {
   padding: 0px;
-  margin: 40px 3px;
-  min-width: 150px;
+  margin: 50px 3px;
```
Comment (Nilvus) [inline data/themes/darktable.css]: 45px would be better. Test a portrait image and see icons and text around to see that (no text superpose). Good on landscape also with 45px.

Comment (TurboGit) [reply data/themes/darktable.css]: Works for me! Thanks.

**Comment by TurboGit** [comment]
Merged.

**Pull request merged**